.isort.cfg

```diff
@@ -19,5 +19,6 @@ multi_line_output=3
 order_by_type=True
 sections=FUTURE,STDLIB,THIRDPARTY,QT,LOCALFOLDER,RESOURCES,PICARD_UI
 skip_glob=**/picard/ui/ui_*.py
-skip=./tagger.py,picard/const/attributes.py,picard/const/countries.py,picard/resources.py,scripts/picard.in
+skip=./tagger.py,picard/const/attributes.py,picard/const/countries.py,picard/resources.py,scripts/picard.in,picard/const/__init__.py,picard/formats/__init__.py
```

Comment:
What's the issue with `formats/__init__.py` and `const/__init__.py`? Shouldn't it be possible to sort the imports to the start, too?

Reply:
I just excluded problematic ones for now, either due to style or just crappy coding like caa providers